tests/cross_functional/kcs/test_monitor_recovery.py

```
@@ -127,93 +130,155 @@ def test_monitor_recovery(
+
+        logger.info(f"Checksums after recovery: {new_md5_sum}")
+        logger.assertion(
+            f"Data integrity check: original={self.md5sum}, "
+            f"after_recovery={new_md5_sum}, "
+            f"match={collections.Counter(new_md5_sum) == collections.Counter(self.md5sum)}"
+        )
         if collections.Counter(new_md5_sum) == collections.Counter(self.md5sum):
-            logger.info(
-                f"Verified: md5sum of {self.filename} on pods matches with the original md5sum"
-            )
+            logger.info(f"Data integrity verified: checksums match for {self.filename}")
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)